### DISCUSSION TITLE
1.1.2

### DISCUSSION THREAD

**Discussion by vicwomg** [vicwomg/pikaraoke#199] [Announcements]
### What's new (1.1.2)

- Translation framework, with Chinese language support. Thanks tbelaire!
- HTML splash screen at /splash for remote browser display of QR code and up next
- Use yt-dlp instead of youtube-dl for faster downloads
- Drop omxplayer from pi install script (no longer in Debian package manager)
- Bugfixes

The 1.1.2 pi image is running Raspberry Pi OS Buster and confirmed working on pi4 and pi3. 

Note that it is not running Bullseye OS, which appears to still be problematic for running vlc on the CLI (see https://github.com/vicwomg/pikaraoke/issues/161 ). I cannot recommend upgrading to Bullseye at this time.

<hr /><em>This discussion was created from the release <a href='https://github.com/vicwomg/pikaraoke/releases/tag/'>1.1.2</a>.</em>

**Comment by coffee-mage** [comment]
1.1.2 image still seems to download via youtube-dl (slow) for me.   I'm not sure if i'm doing something wrong.   Happy to provide any additional details requested.

**Reply by vicwomg** [reply]
I just loaded up a fresh 1.1.2 and can't reproduce this issue. The latest versions of pikaraoke technically can't run youtube-dl, references to it in the youtube-dl binary path in pikaraoke code have been removed. Is it possible it's just slow network connection? Did you try updating the yt-dlp from the menu?

**Reply by coffee-mage** [reply]
Sorry complete brain fart on my part.  I used an older image.  When I use the proper new 1.1.2, everything works as expected.  Thanks for great project.
